keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -102,10 +102,14 @@ See [Post-GA tasks](#post-ga-tasks) for decided out-of-scope features.
 ### Defaulting and pruning for custom resources is implemented
 
 Both defaulting and pruning and also read-only validation are blocked by the
-OpenAPI subset definition (next point). An update of the [old Pruning for
-CustomResources KEP](https://github.com/kubernetes/enhancements/pull/709) and the implementation
-([pruning PR](https://github.com/kubernetes/kubernetes/pull/64558), [defaulting
-PR](https://github.com/kubernetes/kubernetes/pull/63604)), are follow-ups as soon as unblocked.
+OpenAPI subset definition (next point). 
+
+See the update to [Pruning for CustomResources KEP](https://github.com/kubernetes/enhancements/pull/709) 
+and the [Defaulting for Custom Resources KEP](https://github.com/kubernetes/enhancements/pull/1006).
```

Comment:
these are merged, can update to doc links:
* openapi subset: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190425-structural-openapi.md
* pruning: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20180731-crd-pruning.md
* defaulting: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190426-crd-defaulting.md

Reply:
Updated all occurrences in the doc.